server/status/runtime.go

```diff
@@ -47,6 +47,9 @@ var (
 	metaCPUSysNS       = metric.Metadata{Name: "sys.cpu.sys.ns", Help: "Total system cpu time in nanoseconds"}
 	metaCPUSysPercent  = metric.Metadata{Name: "sys.cpu.sys.percent", Help: "Current system cpu percentage"}
 	metaRSS            = metric.Metadata{Name: "sys.rss", Help: "Current process RSS"}
+	metaFDOpen         = metric.Metadata{Name: "sys.fd.open", Help: "Process open file descriptors"}
+	metaFDSoftLimit    = metric.Metadata{Name: "sys.fd.softlimit", Help: "Process open FD soft limit"}
+	metaFDHardLimit    = metric.Metadata{Name: "sys.fd.hardlimit", Help: "Process open FD hard limit"}
```

Comment:
I don't think we need to monitor the hard limit; the soft (i.e. current) limit is all that matters.